SampleNHAppSwiftUI2/SampleNHAppSwiftUI2/Views/TagList.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
Same comment as above, please remove all personal information in the headers and use the Microsoft standard:
```
//----------------------------------------------------------------
//  Copyright (c) Microsoft Corporation. All rights reserved.
//----------------------------------------------------------------
```

Reply:
> @hyounoo Thank you very much for the contribution! I love it!
> 
> Quick question though. Should we then remove the [existing SwiftUI iOS example](https://github.com/Azure/azure-notificationhubs-ios/tree/main/SampleNHAppSwiftUI) then and replace it with this one? Also, should we add a [macOS version](https://github.com/Azure/azure-notificationhubs-ios/tree/main/SampleNHAppMacSwiftUI/SampleNHAppMacSwiftUI) to replace this one as well? I think that's more likely the case since we're focused on iOS 14 on forward.

No, existing sample is targeting iOS13 with SwiftUI v1 and there are plenty of people still implementing their project targeting iOS13.
I think the same goes to macOS too.

So please keep the existing sample and add this new sample for the early adapters.